cmd/blockchaincmd/change_weight.go

```diff
@@ -371,9 +367,6 @@ func changeWeightACP99(
 	if err != nil {
 		return err
 	}
-	if err = signatureaggregator.UpdateSignatureAggregatorPeers(app, network, extraAggregatorPeers, aggregatorLogger); err != nil {
```

Comment:
same here for CreateSignatureAggregatorInstance, or make GetSignatureAggregatorEndpoint to start an aggregator if not available

Reply:
addressed